plugins/outputs/file/file.go

```diff
@@ -18,19 +19,46 @@ import (
 //go:embed sample.conf
 var sampleConfig string
 
+var ValidCompressionAlgorithmLevels = map[string][]int{
+	"zstd": {1, 3, 7, 11},
+	"gzip": {-2, -1, 1, 9},
+	"zlib": {-2, -1, 1, 9},
+}
```

Comment:
Maybe also add identity just in case someone wants it for some reason. It will work just the same.
```suggestion
var ValidCompressionAlgorithmLevels = map[string][]int{
	"zstd": {1, 3, 7, 11},
	"gzip": {-2, -1, 1, 9},
	"zlib": {-2, -1, 1, 9},
	"identity": {0},
}
```